.travis.yml

```diff
@@ -16,6 +18,15 @@ python:
 - '3.6'
 matrix:
   include:
+  - os: osx
+    language: generic
+    env: NMODE=nikola
+    python: '3.6'
+    before_install:
+    - brew upgrade pyenv
```

Comment:
I considered that but it doesn't provide any control over which specific version of python is running and it means that changes in Homebrew could unexpectedly break builds.

The crux of it is that Travis support for Python builds on macOS is a little lacking and this was the simplest, reliable, reproducible way that I could come up with to get things to build (there are a lot of failed attempts squashed into this commit).